### DISCUSSION TITLE
Precompiled Lazer iOS/iPad OS builds available on Github

### DISCUSSION THREAD

**Discussion by cainan-c** [ppy/osu#16274] [Q&A] [answered]
Looking to try out Lazer but the testflight is limited to 10,000 users. Would be nice to see an ipa avaliable for those who know how to sideload applications onto their iOS devices
Would it be possible to have these hosted alongside other platforms?

**Comment by peppy** [comment, marked as answer]
No, but you can build it yourself.

**Reply by scattagain** [reply]
Thanks, Ill just build this with the mac i totally have

**Reply by frenzibyte** [reply]
The game will be published to the App Store soon enough anyways, stay tuned.
